src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -24,6 +11,19 @@
 using Greg.Requests;
 using Prism.Commands;
 using PythonNodeModels;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Forms;
```

Comment:
Did your settings for the sorting got reverted again?

Reply:
Did they? I asked @reddyashish, I wasn't sure if the system ones should be on top or not. I did update Visual Studio recently, perhaps it reset the extension settings. I will re-update and push again, for all affected PRs.

Reply:
I re-saved the file using these settings: 
<img width="528" alt="image" src="https://github.com/DynamoDS/Dynamo/assets/5354594/63d55b3a-ebef-49e4-b9e2-ae294fbbdbab">

Reply:
@dnenov https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#usings. All good now